packages/eslint-config/package.json

```diff
@@ -18,19 +18,21 @@
   },
   "dependencies": {
     "@babel/eslint-parser": "7.25.1",
-    "@typescript-eslint/eslint-plugin": "7.18.0",
-    "@typescript-eslint/parser": "7.18.0",
-    "eslint": "8.57.0",
+    "@eslint/compat": "1.1.1",
```

Comment:
Is this a new dep?

Reply:
ok can you please add a comment about this then so we don't forget to remove it once nextjs adds support for eslint 9 please?